library/src/main/java/com/chuckerteam/chucker/api/internal/ui/MainActivity.java

```diff
@@ -46,10 +51,30 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         setSupportActionBar(toolbar);
         toolbar.setSubtitle(getApplicationName());
 
+        FeatureManager featureManager = new FeatureManager(getApplicationContext());
```

Comment:
Instead of modifying the layout to display stuff in the frame layout, I would modify the HomePageAdapter itself, so that it only displays the one tab is the HIDE_ERROR_TAB is set to true.